app/lib/active_record_utils.rb

```diff
@@ -12,8 +12,9 @@ def self.with_transaction_and_rescue(audit_results)
     rescue ActiveRecord::RecordNotFound => e
       audit_results.add_result(AuditResults::DB_OBJ_DOES_NOT_EXIST, e.inspect)
     rescue ActiveRecord::ActiveRecordError => e
+      Rails.logger.error("db update error; more context to follow: #{e.inspect}: #{e.backtrace.inspect}")
       audit_results.add_result(
-        AuditResults::DB_UPDATE_FAILED, "#{e.inspect} #{e.message} #{e.backtrace.inspect}"
+        AuditResults::DB_UPDATE_FAILED, e.inspect
```

Comment:
- `e.inspect` contains `e.message`, so no need to duplicate.
- removing backtrace here will greatly shorten messages for this audit error
- honeybadger will generate its own stack track